### PR TITLE
integrate mayhem

### PR DESCRIPTION
**Pull request by jbgreenh**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+project: jbgreenh/rapidcsv
```
Comment (xansec) [inline Mayhemfile:1]: Tests has been changed to testsuite. Please also use a generic project name.

File: tests-mayhem/nohdr.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+64.529999,64.800003,64.139999,64.620003,21705200,64.620003
```
Comment (xansec) [inline tests-mayhem/nohdr.csv:1]: This is a fine seed, just keep in mind the tests -> testsuite change of the latest release.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Mayhemfile (hunk 1 above)
Comment (jbgreenh) [reply Mayhemfile]: updated

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**